app/views/user_mailer/admin_hidden_work_notification.html.erb

```diff
@@ -1,12 +1,12 @@
 <% content_for :message do %>
-    <p>Dear <%= @user.default_pseud.byline %>,</p>
+  <% pretty_title = "<i>#{style_link(@work.title.html_safe, work_url(@work))}</i>".html_safe %>
```

Comment:
I need someone else to check me on this, but I'm not sure if we need the `html_safe` on `@work.title`, since we've overridden the [title method](https://github.com/otwcode/otwarchive/blob/17307a8a6294d2af7fab882ea5aa14eb786d51af/app/models/work.rb#L74-L77) to make it always use `html_safe`. 

I tried taking the `html_safe` out and testing with a title that had an ampersand. It looked okay:

```<p>Your work <i><a style="color:#990000" href="http://www.example.com/works/7">Modern & Old-fashioned</a></i> has been reported as being in violation of the Archive of Our Own's <a style="color:#990000" href="http://www.example.com/tos">Terms of Service</a>.</p>```

But it also looked okay when, in addition to removing the `html_safe` call on the title, I removed
- the override on the `title` method
- the `html_safe` at the end of this line
- the `html_safe` on `tos: style_link(t(".tos"), tos_url).html_safe`

It only broke when I finally removed the `html_safe` at the very end of line 5.

tl;dr: we might be being overzealous with our `html_safe` -- it's possible we don't need any on line 2 and we only need the final one on line 5.

Reply:
I'm going to forget if I don't comment now: we should use the style_creation_link helper. https://github.com/otwcode/otwarchive/blob/8d1456a2b876e5d743c6e71748f8b0cb87f6eb1d/app/helpers/mailer_helper.rb#L12-L16